schema/deploy/function_application_application_status.sql

```diff
@@ -0,0 +1,23 @@
+-- Deploy ggircs-portal:function_application_application_status to pg
+-- requires: table_application_status
+-- requires: table_application
+
+begin;
+
+create or replace function ggircs_portal.application_application_status(application ggircs_portal.application)
+    returns ggircs_portal.application_status
+as
+$function$
+declare
+begin
+    return (
+        select row(_application_status.*)
```

Comment:
Do you need `row()` ?

Reply:
Yes, because the query must return a single column